libraries/Adapters/Microsoft.Bot.Builder.Adapters.WeChat/WeChatClient.cs

```diff
@@ -0,0 +1,735 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Bot.Builder.Adapters.WeChat.Schema;
+using Microsoft.Bot.Builder.Adapters.WeChat.Schema.JsonResult;
+using Microsoft.Bot.Builder.Adapters.WeChat.Schema.Response;
+using Microsoft.Bot.Schema;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Logging.Abstractions;
+using Newtonsoft.Json;
+
+namespace Microsoft.Bot.Builder.Adapters.WeChat
+{
+    /// <summary>
+    /// A WeChat client is used to commnicate with WeChat API.
+    /// </summary>
+    public class WeChatClient
+    {
+        private static readonly string APIHost = "https://api.weixin.qq.com";
```

Comment:
Rename to ApiHost to align with capitalization conventions.
This field can be made const instead of static readonly